plugins/scheduled_task/lib/api/types/object_tasks/KalturaModifyEntryObjectTask.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @package plugins.scheduledTask
+ * @subpackage api.objects.objectTasks
+ */
+class KalturaModifyEntryObjectTask extends KalturaObjectTask
```

Comment:
This is actually consistent with other objects https://github.com/kaltura/server/tree/Lynx-12.5.0/plugins/scheduled_task/lib/api/types/object_tasks